desktop/lib/setup.coffee

```diff
@@ -263,7 +263,6 @@ module.exports = (app) ->
   app.use require "../apps/fair"
   app.use require "../apps/fair_info"
   app.use require "../apps/fair_organizer"
-  app.use require "../apps/auction"
   app.use require "../apps/auction2"
```

Comment:
A future PR can fix all of the naming (auction2 --> auction), but I thought that would make this diff a little too much to handle.